ci/kind/test-e2e-kind.sh

```diff
@@ -109,8 +109,8 @@ COMMON_IMAGES_LIST=("registry.k8s.io/e2e-test-images/agnhost:2.29" \
                     "projects.registry.vmware.com/antrea/perftool" \
                     "antrea/antrea-ubuntu:latest" \
                     "antrea/flow-aggregator:latest" \
-                    "projects.registry.vmware.com/antrea/theia-clickhouse-operator:0.18.2" \
-                    "projects.registry.vmware.com/antrea/theia-metrics-exporter:0.18.2" \
+                    "projects.registry.vmware.com/antrea/theia-clickhouse-operator:0.21.0" \
+                    "projects.registry.vmware.com/antrea/theia-metrics-exporter:0.21.0" \
```

Comment:
As we plan to upload images without theia prefix, are we going to remove all things related to theia-clickhouse-operator and theia-metrics-exporter in theia repo in this PR?

Reply:
let's merge https://github.com/antrea-io/theia/pull/312 first, so that we can build new images without theia prefix, then we replace all the image names in this PR